### DISCUSSION TITLE
New user, sanity check on use of straight.el and calling code-review-forge-pr-at-point

### DISCUSSION THREAD

**Discussion by magthe** [wandersoncferreira/code-review#250] [Q&A]
I've just pulled in `code-review` using

```
(use-package code-review)
```

That gave me an error that I didn't quite understand, but when calling `code-review-start` a build of sqlite was triggered and after that it seems installation is slightly happier, but I see the following error on start:

```
⛔ Error (use-package): code-review/:catch: Given parent class emacsql-sqlite-connection is not a class
```

I get the same error message when calling `code-review-forge-pr-at-point`.

Is this something others have encountered, and is there a known workaround?

**Comment by dakra** [comment]
Try with this patch: https://github.com/wandersoncferreira/code-review/pull/246

**Comment by magthe** [comment]
Thanks, @dakra !
